honeybee_radiance/lib/modifiers.py

```diff
@@ -0,0 +1,13 @@
+"""Library of default modifiers for honeybee-radiance."""
+
+from honeybee_radiance.primitive.material import Plastic, Glass
+
+
+generic_exterior_wall = Plastic.from_single_reflectance('generic_ext_wall_0.35', 0.35)
+generic_interior_wall = Plastic.from_single_reflectance('generic_int_wall_0.50', 0.5)
+generic_ceiling = Plastic.from_single_reflectance('generic_ceiling_0.80', 0.8)
+generic_roof = Plastic.from_single_reflectance('generic_roof_0.20', 0.2)
```

Comment:
This one is tricky because as you noted in a single face model ceiling and roof will be the same face. I wish there was a solution to pick two different materials for two different sides of a face but for now I take your suggestion and change it to 0.7